src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -140,6 +141,9 @@ const ColonyHome = ({ match, location }: Props) => {
         </aside>
         <div className={styles.mainContent}>
           <ColonyTotalFunds colony={colony} />
+          <div className={styles.domainsDropdownContainer}>
```

Comment:
![Screenshot from 2020-11-12 17-39-51](https://user-images.githubusercontent.com/1193222/98962186-0f845500-250f-11eb-8d3c-1df92aec8cda.png)

When a domain is selected, we should also see that domain's description above the Colony Funds in the right sidebar.

Either add that in here, or open a new Linear issue to keep track of it _(post UAC, when we have the event data?)_

Reply:
Opened!